scripts/mkcloud

```diff
@@ -582,7 +587,6 @@ function createcloudsnapshot()
         create_snapshot_volume "${cloudvg}/${d}"
     done
     restartcloud
-    wait_for_crowbar_ssh
```

Comment:
This commit is moving the `wait_for_crowbar_ssh` to happen later.  I'm not 100% sure but I *think* that's a good idea because nothing before that point requires ssh, so it means ssh startup can happen in parallel with the other stuff, thereby reducing or eliminating the wait time - is that what you were thinking too?

However, even if I agree with the change, AFAICS it does not belong in the same commit as the ntpd fix, and it certainly should have a commit message which justifies the change :-)